### ISSUE TITLE
ZoomButtonGroup pointer areas should support an arbitrary spacing

### ISSUE THREAD

**Issue by samreid** [phetsims/scenery-phet#650]
From a discussion in https://github.com/phetsims/circuit-construction-kit-common/issues/620, @pixelzoom and I discussed that ZoomButtonGroup pointer areas should support an arbitrary spacing.  At the moment, it looks like it assumes the spacing is 0.

@ariel-phet can you please recommend a priority and developer?

**Comment by pixelzoom** [comment]
I'll take care of this one.

**Comment by pixelzoom** [comment]
Implemented in the above commit, with a demo in scenery-phet Components screen for PlusMinusZoomButtonGroup.  Run with `?screens=3&component=PlusMinusZoomButtonGroup&showPointerAreas` and change the value of `const spacing` to reproduce these tests:

`spacing = 0`, pointer areas are fully shifted:

<img width="279" alt="screenshot_760" src="https://user-images.githubusercontent.com/3046552/102414938-91571a80-3fb4-11eb-9eb3-8c56756a0407.png">

`spacing = 10`, partial shift needed:

<img width="294" alt="screenshot_759" src="https://user-images.githubusercontent.com/3046552/102415036-b8155100-3fb4-11eb-8e34-1a1af832dda3.png">

`spacing = 30`, no shift needed:

<img width="326" alt="screenshot_761" src="https://user-images.githubusercontent.com/3046552/102415057-c19eb900-3fb4-11eb-9a03-96422a3411c0.png">

@samreid want to take a look?

**Comment by samreid** [comment]
Here are the pointer areas without the shift:

![image](https://user-images.githubusercontent.com/679486/102638863-689a6680-4115-11eb-8858-a05f254a05c8.png)

Here are the pointer areas with the shift:

![image](https://user-images.githubusercontent.com/679486/102638900-78b24600-4115-11eb-86f9-14b3451827a4.png)

It seems like the intention is to keep the same shape and just move them, but in some cases it may be desirable to truncate the overlap instead of shifting the entire areas.  I'm not recommending a change for this, just wanted to bring it up for the record.  It seems reasonable to shift the entire areas so they have the same "footprint".  And the effect I described could be achieved by changing the dilations.

I also considered factoring out a function that computes the touch area at once instead of mutating it, but the given implementation seems clearer.

In short, nice work!  Closing.
